### PR TITLE
updating home url and pubspec

### PR DESCRIPTION
**Pull request by peterBrxwn**
For the changes in the pubspec, checkout: https://dart.dev/tools/pub/pubspec#supported-fields

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[Flutter] Try to fix the pubspec url issue](https://trello.com/c/LTp2wTua/1071-flutter-try-to-fix-the-pubspec-url-issue)

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 name: countly_flutter
 description: Countly is an innovative, real-time, open source mobile analytics and push notifications platform.
 version: 23.8.4
-homepage: https://support.count.ly/hc/en-us/articles/360037944212-Flutter
+homepage: https://support.count.ly/hc/en-us/articles/360037944212
+repository: https://github.com/Countly/countly-sdk-flutter-bridge
+issue_tracker: https://github.com/Countly/countly-sdk-flutter-bridge/issues
+documentation: https://support.count.ly/hc/en-us/articles/360037944212
+platforms:
```
Comment (turtledreams) [inline pubspec.yaml:8]: Hi peter! it this platforms part necessary?

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (peterBrxwn) [reply pubspec.yaml:8]: We can omit it but then we're allowing pub.dev to determine the platforms our sdk is available on automatically.

Comment (peterBrxwn) [reply pubspec.yaml:8]: should I remove it?

Comment (turtledreams) [reply pubspec.yaml:8]: Awesome, if it is useful it should stay 👍

**Review by turtledreams — APPROVED** [review]

**Review by ijunaid — APPROVED** [review]

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**